client/components/badge/style.scss

```diff
@@ -0,0 +1,22 @@
+.badge {
+	display: inline-block;
+	border-radius: 100px;
+	padding: 2px 11px;
+	text-align: center;
```

Comment:
I might be missing a case that needs this, but I'm not sure if this is necessary.

Reply:
You are right, I think we could live without it.